### PR TITLE
update observations

### PR DESCRIPTION
**Pull request by czengg**

### CODE REVIEW

File: src/models/observations.js [hunk 1]
```diff
@@ -1,19 +1,59 @@
...
-  link: 'link',
   created: new Date(),
   name: '',
   notes: '',
   observedBy: 'Tú',
   attachments: [],
-  icon: null,
-  categoryId: '0',
-  fields: [],
+  categoryId: '',
```
[the remark this answers is not in the document]
Comment (czengg) [reply src/models/observations.js]: They are stored under the tags field

File: src/models/observations.js [hunk 2]
```diff
@@ -1,19 +1,59 @@
...
+  const keep = { created: true };
+  Object.keys(tags).forEach(property => {
+    if (!keep[property] && tags[property] === defaultObservation[property]) {
+      delete tags[property];
+    }
+  });
+
+  // remove topLevel properties
+  const topLevel = ['lat', 'lon', 'attachments', 'id'];
+  topLevel.forEach(p => delete tags[p]);
```
Comment (czengg) [inline src/models/observations.js]: This is where we separate the toplevel properties from those we're keeping in tags. The property in tags are essentially everything that isn't top level and in the observation model, ie: created, categoryId, fields, etc.

**Comment by czengg** [comment]
@gmaclennan this should address bringing the answers for presets up to observation.tags

File: src/models/observations.js [hunk 3]
```diff
@@ -1,19 +1,72 @@
...
+
+  return {
+    lat: lat || 0,
+    lon: lon || 0,
+    version,
+    attachments: (attachments || []).map(id => ({ id })),
+    type: 'observation',
+    device_id: '1',
```
Comment (karissa) [inline src/models/observations.js]: What is device_id?

**Pull request merged**